api-test/src/main/java/io/mosip/testrig/apirig/signup/testrunner/MosipTestRunner.java

```diff
@@ -108,9 +108,13 @@ public static void main(String[] arg) {
 				AdminTestUtil.createEditAndPublishPolicy();
 				PartnerRegistration.deviceGeneration();
 
+				BaseTestCase.domain=".mosip.net";
```

Comment:
why is this  line required,  when you are already handling the default value in line 137